public/renderer.js

```diff
@@ -557,31 +557,67 @@ function getSectorBoundsAllBus (roomInSector) {
 }
 
 function makeRespawnSectorWall (room, borderSide, decayTime) {
-  let x, y
+  let x, y, x2, y2
+  // Edges
   for (let i = 0; i < 50; i++) {
     switch (borderSide) {
       //borderSide is related to the sector side not the room side
       case 'left':
-        x = 49
+        x = 30
         y = i
         break
       case 'right':
-        x = 0
+        x = 20
         y = i
         break
       case 'top':
         x = i
-        y = 49
+        y = 30
         break
       case 'bottom':
         x = i
-        y = 0
+        y = 20
         break
     }
     if (x !== undefined && y !== undefined) {
       room.objects.push({type: 'constructedWall', room: room.room, x, y, decayTime: {timestamp: decayTime}})
     }
   }
+  // Corners
+  for (let i = 0; i < 20; i++) {
+    switch (borderSide) {
+      case 'bottomLeft':
+        x = 30
+        y = i
+        x2 = 30 + i
+        y2 = 20
+        break
+      case 'topLeft':
+        x = 30 + i
+        y = 30
+        x2 = 30
+        y2 = 30 + i
+        break
+      case 'bottomRight':
+        x = i
+        y = 20
+        x2 = 20
+        y2 = i
+        break
+      case 'topRight':
+        x = i
+        y = 30
+        x2 = 20
+        y2 = 30 + i
+        break
+    }
+    if (x !== undefined && y !== undefined && x < 50 && y < 50) {
+      room.objects.push({type: 'constructedWall', room: room.room, x, y, decayTime: {timestamp: decayTime}})
+    }
+    if (x2 !== undefined && y2 !== undefined && x2 < 50 && y2 < 50) {
+      room.objects.push({type: 'constructedWall', room: room.room, x2, y2, decayTime: {timestamp: decayTime}})
+    }
```

Comment:
//fill in missing hole
    if (borderSide === 'bottomRight') {
      room.objects.push({type: 'constructedWall', room: room.room, x: 20, y: 20, decayTime: {timestamp: decayTime}})
    }